src/app.js

```diff
@@ -59,6 +58,20 @@ JoystickLib.getJoystick()
 					return { x: -1, y: 0 };
 				case 'right':
 					return { x: 1, y: 0 };
+				case 'click':
```

Comment:
:+1:

Reply:
Because we have the checking in the other part below, I think we can remove this, it was only useful as an example really.